### DISCUSSION TITLE
Testing PhpUnit read Environment variable definition from .env

### DISCUSSION THREAD

**Discussion by rv2931** [symfony/symfony#46686] [Q&A]
Hello
I'm beginning with Symfony testing and phpunit. I'm trying to make some KernelTestCase. I'm facing the problem of KERNAL_CLASS not defined, I solved it with overloading getKernelClass function, Now I'm facing the problem of DATABASE_URL not defined...
So I'm wondering : is there a way to make phpunit using standard .env .env.test and .local equivalents ? instead of writing in phpunit.xml/phpunit.xml.dist ? I really don't like the idea of writing/overloading something in vendor directory that doesn't follow the standard wya of managing parameters/env vars in symfony
Thank you for your answers ;)
RV

**Comment by ghost** [comment]
You should write a bootstrap file for the tests where you load .env files with Dotenv Component. See for example here https://github.com/symfony/recipes/blob/main/phpunit/phpunit/9.3/tests/bootstrap.php
